lib/tasks/serve.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Mix.Tasks.Serve do
+  use Mix.Task
+
+  def run([port_str]) do
```

Comment:
That's silly, Heroku are all about env. Why is port different? Are we sure it's not both in $PORT, and an env variable?

Failing that...

```bash
PORT=$PORT bin/whatever.sh
```

???